lib/session/call-session.js

```diff
@@ -622,8 +622,11 @@ class CallSession extends Emitter {
       const res = await this.dlg.request({
         method: 'INFO',
         headers: {
-          'X-Reason': 'pauseCallRecording'
-        }
+          'X-Reason': 'pauseCallRecording',
+          ...(this.recordOptions.headers && {'Content-Type': 'application/json'})
```

Comment:
are we guaranteed that this.recordOptions will exist?

Reply:
https://github.com/jambonz/jambonz-feature-server/blob/main/lib/session/call-session.js#L549

before executing recording command, recordOptions = data from recording command. so it always available for recording command